ms2query/create_sqlite_database.py

```diff
@@ -31,37 +34,99 @@ def make_sqlfile_wrapper(sqlite_file_name: str,
-        the number of inchikeys in csv_file_with_inchikey_order.
-    pickled_file_name:
-        This file is expected to contain a list of matchms.Spectrum.
-    csv_file_with_inchikey_order:
-        This csv file is expected to contain inchikeys on the second column
-        starting from the second row. These inchikeys should be ordered in the
-        same way the tanimoto scores are ordered in the npy_file_location.
+    tanimoto_scores_pickled_dataframe_file:
+        A pickled file with tanimoto scores. The column names and indexes are
+        14inchikeys.
+    list_of_spectra:
+        A list with spectrum objects
+    columns_dict:
+        Dictionary with as keys columns that need to be added in addition to
+        the default columns and as values the datatype. The defaults columns
+        are spectrum_id, peaks, intensities and metadata. The additional
+        columns should be the same names that are in the metadata dictionary,
+        since these values will be automatically added in the function
+        add_list_of_spectra_to_sqlite.
+        Default = None results in the default columns.
+    progress_bars:
+        If progress_bars is True progress bars will be shown for the different
+        parts of the progress.
+    spectrum_id_column_name:
+        The spectrum id column name is the name given to the column storing the
+        spectrum ids. This is important since this name will be used to look
+        up the spectrum id in the metadata. Per version of the data this
+        differs between 'spectrum_id' and 'spectrumid'
     """
+    # pylint: disable=too-many-arguments
+    add_tanimoto_scores_to_sqlite(sqlite_file_name,
+                                  tanimoto_scores_pickled_dataframe_file,
+                                  progress_bars=progress_bars)
+    create_table_structure(sqlite_file_name,
+                           additional_columns_dict=columns_dict,
+                           spectrum_column_name=spectrum_id_column_name)
+    add_list_of_spectra_to_sqlite(sqlite_file_name,
+                                  list_of_spectra,
+                                  progress_bar=progress_bars)
+
+
+def add_tanimoto_scores_to_sqlite(sqlite_file_name: str,
+                                  tanimoto_scores_pickled_dataframe_file: str,
+                                  temporary_tanimoto_file_name: str
+                                  = "temporary_tanimoto_scores",
+                                  progress_bars: bool = True):
+    """Adds tanimoto scores and inchikeys to sqlite table
+
+    sqlite_file_name:
+        Name of sqlite_file that should be created, if it already exists the
+        tables are added. If the tables in this sqlite file already exist, they
+        will be overwritten.
+    tanimoto_scores_pickled_dataframe_file:
+        A pickled file with tanimoto scores. The column names and indexes are
+        14inchikeys.
+    temporary_tanimoto_file_name:
```

Comment:
You added a #todo elsewhere and I agree: we should alter make use of a different way of temporary storage here.
-- > Make another issue for this (and here leave it as it is).

Reply:
I added an issue: #68